src/kOS/Suffixed/VectorRenderer.cs

```diff
@@ -475,8 +477,12 @@ public void RenderColor()
 
             if (line != null && hat != null)
             {
-                line.SetColors(c1, c2); // The line has the fade effect
-                hat.SetColors(c2, c2);  // The hat does not have the fade effect.
+                // The line has the fade effect from color c1 to color c2:
+                line.startColor = c1; 
+                line.endColor = c2;
+                // The hat does not have the fade effect, staying at color c2 the whole way:
+                hat.startColor = c2;
+                hat.endColor = c2;
```

Comment:
Fixes Warnings: obsoleted method - Setcolors()

Unity changed the API a bit here.